fe1-web/__tests__/model/network/FromJson.ts

```diff
@@ -4,16 +4,17 @@ import {
   ActionType, CloseRollCall,
   CreateLao,
   CreateMeeting, CreateRollCall,
-  ObjectType, OpenRollCall,
+  ObjectType, OpenRollCall, ReopenRollCall,
   StateLao, StateMeeting,
   UpdateLao, WitnessMessage
-} from '../../../Model/Network/Method/Message/data';
-import { getStorageCurrentLao, initialise } from '../../../Store/Storage';
-import { ProtocolError } from '../../../Model/Network';
-import { Base64Data, Hash, Lao, PrivateKey, PublicKey, Timestamp } from '../../../Model/Objects';
+} from 'Model/Network/Method/Message/data';
+import { initialise } from 'Store/Storage';
```

Comment:
it might be worth aliasing this for clarity, like `as storeInitialize`

also, we might want to rename `initialise` to `initialize`, as american english is more common in code bases :)